pkg/internal/apis/certmanager/types_issuer.go

```diff
@@ -177,6 +177,10 @@ type VaultIssuer struct {
 	// "my_pki_mount/sign/my-role-name".
 	Path string
 
+	// Name of the vault namespace. Namespaces is a set of features within Vault Enterprise that allows Vault environments to support Secure Multi-tenancy. e.g: "ns1"
+	// More about namespaces can be found here https://www.vaultproject.io/docs/enterprise/namespaces
```

Comment:
Can you add this line into all API versions? It is super useful but the internal API comments are not exposed

Reply:
Added this to all versions.